stacks/portstack.go

```diff
@@ -376,29 +376,41 @@ func (ps *PortStack) HandleEth(dst []byte) (n int, err error) {
 //
 // If a handler returns any other error the port is closed.
 func (ps *PortStack) handleEth(dst []byte) (n int, err error) {
+	//this would be much better named as getNextOutboundPacket
+
+	//println ("portStack:handleEth ",string(dst)) //@@@REMOVE
+
```

Comment:
This discussion belongs in github issue tracker! also remove the comment

Reply:
Opened issue .. I do think it needs a better name - it was one area that was very opaque to me for a long time, the fact the slice is passed by refence (for population) is not super obvious when coming from other languages